vision/cloud-client/detect/set_endpoint.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+def set_endpoint():
+    """Change your endpoint"""
+    # [START vision_set_endpoint]
+    from google.cloud import vision
+
+    client_options = {'api_endpoint': 'eu-vision.googleapis.com:443'}
+
+    client = vision.ImageAnnotatorClient(client_options=client_options)
+    # [END vision_set_endpoint]
+    image_source = vision.types.ImageSource(
+        image_uri='gs://cloud-samples-data/vision/text/screen.jpg')
+    image = vision.types.Image(source=image_source)
+
+    response = client.text_detection(image=image)
+
+    print('Texts:')
+    for text in response.text_annotations:
+        print('{}'.format(text.description))
+
+        vertices = (['({},{})'.format(vertex.x, vertex.y)
```

Comment:
Nit: no need for the extra parentheses since the square brackets from the list comprehension already "serve as the parentheses". The parser knows that a `[` opened and can be closed at any other line just like a `(`.